Win32Emu/Win32/Modules/DDrawModule.cs

```diff
@@ -842,18 +842,6 @@ private uint DDraw_CreatePalette(ICpu cpu, VirtualMemory memory)
 		/// This is kept for compatibility with non-WASM platforms.
 		/// On WASM, this should not be called - use DDraw_CreateSurfaceAsync instead.
 		/// </summary>
-		private uint DDraw_CreateSurface(ICpu cpu, VirtualMemory memory)
```

Comment:
Upon removing this function, did you check if there are any regressions in the Async function? Like, is there something in here the Async version doesn't have?